pytorch_lightning/plugins/training_type/sharded.py

```diff
@@ -54,7 +67,7 @@ def _reinit_optimizers_with_oss(self):
                 optim_class = type(optimizer)
                 zero_optimizer = OSS(params=optimizer.param_groups, optim=optim_class, **optimizer.defaults)
                 if _FAIRSCALE_OSS_FP16_BROADCAST_AVAILABLE:
-                    is_fp16 = self.lightning_module.trainer.precision == 16
+                    is_fp16 = self.lightning_module.trainer.precision == "mixed"
```

Comment:
wasn't that properly handled before?

Reply:
yeah, unfortunately, will separate this small fix.